services/ymax-planner/package.json

```diff
@@ -28,10 +33,16 @@
   "author": "Agoric",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@graphql-codegen/cli": "^6.0.1",
+    "@graphql-codegen/client-preset": "^5.1.1",
+    "@graphql-codegen/schema-ast": "^5.0.0",
+    "@graphql-codegen/typescript-graphql-request": "patch:@graphql-codegen/typescript-graphql-request@npm%3A6.3.0#~/.yarn/patches/@graphql-codegen-typescript-graphql-request-npm-6.3.0-38043fd4dc.patch",
```

Comment:
What do we lose by omitting this patch? I suspect it's not worth the maintenance cost.

Have you filed it upstream?

Reply:
Yes; there's actually a cluster of them. Without these patches we'd lose support for the GraphQL operation descriptions, and on balance I think they're worth keeping at least until the burden is actually too great.
* https://github.com/ardatan/graphql-tools/issues/7658
* https://github.com/dotansimha/graphql-code-generator-community/issues/1272
* https://github.com/dotansimha/graphql-code-generator/issues/10480